src/common/constants.js

```diff
@@ -101,7 +101,7 @@ export const DISEASE_AREAS = [
   'Osteosarcoma',
   'Structural Heart & Other Defects',
   'Syndromic Cranial Dysinnervation Disorders',
-  'CBTTC/PNOC Brain Tumor Dataset',
+  'Pediatric Brain Tumors',
```

Comment:
I personally prefer `text-transform: capitalize;` for this stuff, what do you think?

Reply:
An issue with `text-transform: capitalize` is that we end up with `Select An Option` instead of `Select an option` for a default.

Or, more concretely, if say the third item needs to be `Patients with both childhood....`instead of `Patients With Both Childhood`. We have this central file to specify vs a bunch of CSS rules to target specific children across potentially multiple select forms.